static/js/ProfileList.tsx

```diff
@@ -38,6 +45,44 @@ const ProfileList: FC = () => {
       </div>
     );
 
+    const touchesNetwork = Object.values(profile.devices).some(
+      (device) => device.type === "nic"
+    );
+    const touchesStorage = Object.values(profile.devices).some(
+      (device) => device.type === "disk"
+    );
+    const touchesCloudInit = Object.keys(profile.config).some((config) =>
+      config.startsWith("cloud-init")
+    );
+    const touchesLimits = Object.keys(profile.config).some((config) =>
+      config.startsWith("limits")
+    );
+
+    const effects = (
+      <>
+        {touchesNetwork && (
+          <Tooltip message="Network" position="btm-center">
+            <Icon name="connected" />
+          </Tooltip>
+        )}
+        {touchesStorage && (
+          <Tooltip message="Storage" position="btm-center">
+            <Icon name="pods" />
+          </Tooltip>
+        )}
+        {touchesCloudInit && (
+          <Tooltip message="Cloud init" position="btm-center">
+            <Icon name="restart" />
+          </Tooltip>
+        )}
+        {touchesLimits && (
+          <Tooltip message="Limit resources" position="btm-center">
+            <Icon name="priority-low" />
+          </Tooltip>
+        )}
```

Comment:
The icons look a bit too close to each other - should we set some (even little) spacing between them?

Reply:
Yeah, I added an inline List as a wrapper to add some spacing.